src/read.rs

```diff
@@ -401,6 +401,17 @@ impl<R> Reader<R>
 
 }
 
+impl Reader<std::fs::File> {
+
+    /// Opens a file at the given location
+    ///
+    /// This is a convenience wrapper of the `new` function.
+    pub fn from_path(path: &std::path::Path) -> Result<Self, ReadError> {
```

Comment:
It looks like this might be a less efficient version of [this method](https://github.com/est31/audrey/blob/a7844eef4dd2a8d6f0db9a6813d500d62ed0534d/src/read.rs#L190) - do you mind if we remove this in favour of the existing impl?

Reply:
Oh right... Sorry, will fix.